code_generation/quantities/output.py

```diff
@@ -7,6 +7,8 @@
 base_muons_mask = Quantity("base_muons_mask")
 good_muons_mask = Quantity("good_muons_mask")
 veto_muons_mask = Quantity("veto_muons_mask")
+veto_muons_mask_2 = Quantity("veto_muons_mask_2")
+veto_muons_mask_combined = Quantity("veto_muons_mask_combined")
```

Comment:
Where is `veto_muons_mask_combined` used?

Reply:
upps, a remnant :wink: